src/wazuh_modules/inventory_harvester/src/fimInventory/fimContext.hpp

```diff
@@ -763,6 +763,20 @@ struct FimContext final
         return m_keySanitized;
     }
 
+    std::string_view elementType() const
+    {
+        switch (m_originTable)
+        {
+            case OriginTable::File: return "file";
+
+            case OriginTable::RegistryKey: return "registry_key";
+
+            case OriginTable::RegistryValue: return "registry_value";
+
+            default: return "invalid";
+        }
+    }
+
```

Comment:
I saw you asked about including this even tho was not in the requirements. It's the only one missing, I already check with Gaby the timestamp field.